.gitignore

```diff
@@ -46,3 +46,6 @@ jspm_packages
 # Ignores all index.js files for all components since they are auto-generated.
 **/index.js
 !/index.js
+
+
+package-lock.json
```

Comment:
You probably have a reason for ignoring this file, but I think you are supposed to commit it to source control so that updates to dependencies only happen explicitly when you update them.

Reply:
Yes, this should indeed be committed. Could avoid potential issues with libraries that don't respect semver.